great_expectations/render/renderer/site_builder.py

```diff
@@ -673,25 +673,23 @@ def _get_call_to_action_buttons(self, usage_statistics):
         """
         create_expectations = CallToActionButton(
             "How to Create Expectations",
-            # TODO update this link to a proper tutorial
-            "https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations.html",
+            "https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_create_expectations",
         )
         see_glossary = CallToActionButton(
             "See More Kinds of Expectations",
             "https://greatexpectations.io/expectations",
         )
         validation_playground = CallToActionButton(
             "How to Validate Data",
-            # TODO update this link to a proper tutorial
-            "https://docs.greatexpectations.io/en/latest/guides/how_to_guides/validation.html",
+            "https://docs.greatexpectations.io/docs/tutorials/getting_started/tutorial_validate_data",
```

Comment:
Same here. I'm on the fence about whether we want to link to the geting started tutorial here, or instead an equivalent how-to guide. https://docs.greatexpectations.io/docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint

Reply:
For this change and the one above: In the Universal Map ToC update I'm adding index pages to the top level folder in the ToC for each step's how to guides, so at that point we can link to a group of how to guides by having the link point to a subsection of the index page.  That might be the best answer if there is more than one way to do a thing, but it won't be available until that update merges into develop.

Reply:
After looking at it a little more, I think that after the ToC update comes out we should have these point to the overview pages for the Create Expectations and the Validate Data steps.  I'll add a subtask into my updates to do that when they are ready.

Reply:
For now, I would lean more toward using the links that @talagluck proposed.  Those are discrete how to guides, while the tutorial pages are parts of the getting started tutorial and not really meant to be referenced outside of that context.